src/test/java/teammates/test/cases/ui/InstructorFeedbackAddActionTest.java

```diff
@@ -37,149 +39,172 @@ public void testExecuteAndPostProcess() throws Exception{
         //TODO make sure IFAA does assertNotNull for required parameters then uncomment
         //verifyAssumptionFailure(Const.ParamsNames.COURSE_ID, instructor1ofCourse1.courseId);
         
+        
         ______TS("Typical case");
         
-        String[] params =
-                createParamsCombinationForFeedbackSession(
-                        instructor1ofCourse1.courseId, "ifaat tca fs", 0);
+        String[] params = createParamsCombinationForFeedbackSession(
+                instructor1ofCourse1.courseId, "ifaat tca fs", 0);
```

Comment:
can be indented further so that the 2nd line starts on the right side of the parent element.
```
String[] params = createParamsCombinationForFeedbackSession(
                               instructor1ofCourse1.courseId, "ifaat tca fs", 0);
```